src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/controllers/OcrValidationControllerTest.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.hmcts.reform.bulkscanccdeventhandler.controllers;
+
+import com.google.common.base.Charsets;
+import com.google.common.io.Resources;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.io.IOException;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(OcrValidationController.class)
+public class OcrValidationControllerTest {
```

Comment:
would be nice to add test cases for scenarios like ocr fields with some fields missing(say out of 5 fields which you expect 2 are missing)

Reply:
I didn't include OCR form validation in this PR. I'll add more test scenarios in the next PR.
This PR is only for the controller

Reply:
I didn't include OCR form validation in this PR. I'll add more test scenarios in the next PR.
This PR is only for the controller